assignment/core/src/main/resources/OSGI-INF/config/CryptoConfiguration.dev.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          jcr:primaryType="sling:OsgiConfig"
+          publicKey="openssl genpkey -algorithm RSA -out stage_private_key.pem -pkeyopt rsa_keygen_bits:2048
+LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0NCk1JSUJJakFOQmdrcWhraUc5dzBC
+QVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBcmZpRlhHK3BpejdQYU02dVlPd1YNCmJw
+WDFWWE0xc3o0NWtmMGJIbUxSb2JwNFdFc0lOWk1CbmtER1lwOHQzaVhzem1BWTh0
+Y1ozbDRZVVdadUdMaWENCjZDR2FUZ2NYZjhlajYzS1RKa2UvdmphTjE3ejhxNFZy
+Y0pTVitPWW1yTUxHeGhhYVJhcWNPYWowZjNXcmRFQ2oNCkNpV2FOeWNmSFZ0WkNF
+SFRtNXNIRS9PZlhJQkNHNTI2aDUwcUgwQjVkSXZnZnIzV1Y0Z2ErOHRucjlwR0hB
+YzQNCmppTitTSFJUNVNndEhCcTVybnFqaHNFc0syLzREL2Q2K2JTaGJhK1daNGY1
+S0FWT1g0NTVEOGZWRFNBQkpXdTANCmI5d21sOFRVYjFKRCtNajdUdHc0dkhiZXZp
+TnpucFVFTGh0UmNESys3QmdONWY5SUhFTmJXVVRtM0hUQmtJb20NCkZRSURBUUFC
+DQotLS0tLUVORCBQVUJMSUMgS0VZLS0tLS0NCg==
+"
+          privateKey="LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tDQpNSUlFdmdJQkFEQU5CZ2txaGtp
```

Comment:
Public and private keys are sensitive credentials, they should never be checked into version control

Reply:
done removed